sesman/chansrv/clipboard.c

```diff
@@ -372,6 +378,16 @@ clipboard_init(void)
         return 0;
     }
 
+    if (0 != config_read(&g_cfg)) {
```

Comment:
Turned out to be not as easy this in xrdp (at least for me), this solution is clearer and cuts the clipboard flow at the source.
About reading the sesman conf, I would argue that chansrv is part of sesman, it does not seem inappropriate.